### ISSUE TITLE
Docker build fails with "/Cargo.lock" not found

### ISSUE THREAD

**Issue by paulmis** [AntoniosBarotsis/qr-rs#1]
Docker build fails on Windows 10 when following the `README` instructions:
```
C:\Users\jesuschrist\source\repos\qr-rs>docker build -t qr-rs .
[+] Building 0.7s (11/20)
 => [internal] load build definition from Dockerfile                                                                                       0.0s
 => => transferring dockerfile: 32B                                                                                                        0.0s
 => [internal] load .dockerignore                                                                                                          0.0s
 => => transferring context: 2B                                                                                                            0.0s
 => [internal] load metadata for docker.io/library/rust:1.64-alpine                                                                        0.5s
 => CACHED [stage-1 1/2] WORKDIR /opt/qr-rs                                                                                                0.0s
 => [internal] load build context                                                                                                          0.0s
 => => transferring context: 280B                                                                                                          0.0s
 => CANCELED [builder  1/14] FROM docker.io/library/rust:1.64-alpine@sha256:382f935713e54efb4dd6347bb3af467764d6d3b590ee19633c8a4e23fda41  0.1s
 => => resolve docker.io/library/rust:1.64-alpine@sha256:382f935713e54efb4dd6347bb3af467764d6d3b590ee19633c8a4e23fda4116a                  0.0s
 => => sha256:382f935713e54efb4dd6347bb3af467764d6d3b590ee19633c8a4e23fda4116a 547B / 547B                                                 0.0s
 => => sha256:9e8c9a67a08508d83adca49fd76e756edbc625fc3e1ffc6e0eb48831be7c3bde 953B / 953B                                                 0.0s
 => => sha256:0aa9cde1d30f2d9a3312d94df8d6764caccd6d9740c39cf7ecb16561615b8cbf 4.00kB / 4.00kB                                             0.0s
 => CACHED [builder  2/14] RUN apk add --no-cache musl-dev                                                                                 0.0s
 => CACHED [builder  3/14] WORKDIR /opt                                                                                                    0.0s
 => CACHED [builder  4/14] RUN cargo new --bin qr-rs                                                                                       0.0s
 => CACHED [builder  5/14] WORKDIR /opt/qr-rs                                                                                              0.0s
 => ERROR [builder  6/14] COPY ./Cargo.lock ./Cargo.lock                                                                                   0.0s
------
 > [builder  6/14] COPY ./Cargo.lock ./Cargo.lock:
------
failed to compute cache key: "/Cargo.lock" not found: not found
```

**Comment by AntoniosBarotsis** [comment]
Ah my mistake, ignoed the lockfile out of habit because it gets generated locally after you build, should work fine once I add it back again

**Comment by AntoniosBarotsis** [comment]
Fixed in ccdb887c7cb0a347b0554e1d825aa4078542f3d6.
